### DISCUSSION TITLE
compact_target

### DISCUSSION THREAD

**Discussion by Lich-2016** [nervosnetwork/ckb#2414] [General]
get_block_template 中compact_target的长度只有4个byte,  校验hash时powHash = eaglesonghash(headerHashWithNonce)， 用来和powhash比较的block target怎么得到？

**Comment by Lich-2016** [comment]
compat_target=0x1a0126d1
那么 target = 0x0126d1 * 2^(0x08 * (0x1a - 0x03)) ？

**Comment by doitian** [comment]
https://github.com/nervosnetwork/ckb/wiki/Header-%C2%BB-compact_target

**Comment by Lich-2016** [comment]
谢谢
